DNN Platform/Website/Resources/Shared/scripts/dnn.searchBox.js

```diff
@@ -38,9 +38,9 @@
             var markup = '<div class="dnnSearchBox">';
             var advancedEnabled = this.options.showAdvanced && this.options.advancedId && $('#' + this.options.advancedId).length;
             if (advancedEnabled) {
-                markup += '<span class="dnnSearchBox_advanced_query" /><a class="dnnSearchBoxClearAdvanced"></a>';
+                markup += '<span class="dnnSearchBox_advanced_query"></span><a class="dnnSearchBoxClearAdvanced"></a>';
             }
-            markup += '<input id="' + this.options.id + '_input" type="text" autocomplete="off" aria-label="Search" />' +
+            markup += '<input id="' + this.options.id + '_input" type="text" autocomplete="off" aria-label="Search"></input>' +
```

Comment:
This change needs to be made, even though `<input />` is self-closing?

Reply:
Yes, that is the main breaking change with jQuery 3.5.0, templates no longer support self closing tags

Reply:
Described in https://jquery.com/upgrade-guide/3.5/

Reply:
Well actually it's not that it is no longer supported, it's just that the behaviour has changed and the recommendation is to not use self closing tags for elements that may have content in them. in this instance, I guess just the span could solve it, but well, it also does not hurt for the input...

Reply:
In this case I'd prefer for `input` to remain self-closing, since it can't have children.  `<input></input>` isn't really valid.
```suggestion
            markup += '<input id="' + this.options.id + '_input" type="text" autocomplete="off" aria-label="Search" />' +
```

Reply:
I agree, done, well at the same time as you actually :)